minecraft-plugin/src/main/resources/config.yml

```diff
@@ -0,0 +1,6 @@
+defaultMaxHearDistance: 100
+defaultNoAttenuationDistance: 10
+defaultLinearAttenuationFactor: 0.9
```

Comment:
IMO we could drop the `default` prefix in all of these properties. I don't get it. Are there some other values that can overwrite these ones?